src/ROUTES.ts

```diff
@@ -576,6 +576,10 @@ const ROUTES = {
         route: 'settings/workspaces/:policyID/tags/edit',
         getRoute: (policyID: string) => `settings/workspaces/${policyID}/tags/edit` as const,
     },
+    WORKSPACE_TAG_EDIT: {
+        route: 'workspace/:policyID/tag/:tagName/edit',
```

Comment:
We're updated these routes
```suggestion
        route: 'settings/workspaces/:policyID/tag/:tagName/edit',
```